libs/core/src/lib/menu/menu.component.ts

```diff
@@ -187,6 +191,7 @@ export class MenuComponent implements MenuInterface, AfterContentInit, AfterView
     ngAfterViewInit(): void {
         this._listenOnMenuMode();
         this._menuService.setMenuMode(this.mobile);
+        this._menuService.sendSelected(this.menuItems.first);
```

Comment:
I think there will be cases when we may want to select a different menu item on first load

Reply:
Default behavior is to have the first element as the title

Reply:
Do you realize that you are changing the behavior of all menus, not only the ones inside `SplitButton`?
There is no reason for any menu to emit its first item as `selected` just after the component has been created.